lib/expand.js

```diff
@@ -1056,7 +1080,7 @@ class Expander {
         // It's an IncludesTypeConstraint.  Create a new Value to represent it.
         return new models.IdentifiableValue(ictCst.isA).withCard(ictCst.card);
       }
-      return;
+      //return;
```

Comment:
If we really don't need this `return` statement we should delete it instead of commenting it out.